### PR TITLE
feat: tooltip styles

### LINKED ISSUE #199: Apply tooltip designs
1 grid view

### PR DESCRIPTION
**Pull request by cazala**
Closes #199 

screen shot 2019-02-19 at 6 39 35 pm

### CODE REVIEW

File: src/components/Modals/ShortcutsModal/ShortcutsModal.tsx [hunk 1]
```diff
@@ -47,11 +47,11 @@ export const renderCombination = (shortcut: ShortcutCombination) => {
   return out
 }
 
-export const renderSimple = (shortcut: SimpleShortcut) => {
+const renderSimple = (shortcut: SimpleShortcut) => {
   return <Chip text={mapLabel(shortcut.value)} key={shortcut.value} />
 }
 
-export const renderAlternative = (shortcut: ShortcutAlternative, onlyFirst: boolean = false) => {
+const renderAlternative = (shortcut: ShortcutAlternative, onlyFirst: boolean = false) => {
```
Comment (nicosantangelo) [inline src/components/Modals/ShortcutsModal/ShortcutsModal.tsx]: We can remove `onlyFirst` here!

```js
  if (onlyFirst) {
    const item = alternatives[0]
    if (item.type === 'combination') {
      out = [...out, ...renderCombination(item)]
    } else {
      out.push(renderSimple(item))
    }
  }
```

File: src/components/ShortcutTooltip/ShortcutTooltip.tsx [hunk 2]
```diff
@@ -1,35 +1,81 @@
...
+
 import { Props } from './ShortcutTooltip.types'
 import './ShortcutTooltip.css'
 
+const renderCombination = (shortcut: ShortcutCombination) => {
+  let out: string[] = []
+
+  for (let i = 0; i < shortcut.value.length; i++) {
+    const label = mapLabel(shortcut.value[i])
+    out.push(label.length === 1 ? label.toUpperCase() : label)
```
Comment (nicosantangelo) [inline src/components/ShortcutTooltip/ShortcutTooltip.tsx]: If you think it's worth it, we can re-use `renderSimple` here, something like:
```js
out.push(renderSimple({ type: 'simple', value: shortcut.value[i], title: shortcut.title }))
```

Same goes for [ShortcutModal](https://github.com/decentraland/builder/pull/202/files#diff-ac850bf1309f5f581c31c53143a70696R33)

**Comment by belohlavek** [comment]
We need to review the homepage tooltip for contest right? @nicosantangelo

**Comment by nicosantangelo** [comment]
> We need to review the homepage tooltip for contest right? @nicosantangelo

Yes! but we can do it after merging this one. Maybe adding an issue blocked by this one?

**Comment by cazala** [comment]
@nicosantangelo I added an abstract class for the repeated code. Thanks for the review!

**Pull request merged**